### ISSUE TITLE
Allow to share twitch stream from Kore to Kodi

### ISSUE THREAD

**Issue by lyndametref** [xbmc/Kore#775]
## Feature suggestion
Allow to share twitch stream from Kore to Kodi. Currently Yatse allows to share a Twitch stream to Kodi, but it does not work with Kore. 

### Context

I mostly choose what I want to see on Kodi on my mobile phone and then send it to Kodi through the share feature of Kore, for example for youtube video. Another source of vide for me is Twitch and I would like to be able to proceed the same way when wanting to watch with my family. I think than any twitch watcher who also use Kore/Kodi could benefit from this feature.

### Additional context, screenshots or links
Here are some relevant links or screenshots:

Other issues reporting this behavior:

- https://github.com/anxdpanic/Twitch-on-Kodi/issues/382
- https://github.com/anxdpanic/Twitch-on-Kodi/issues/423

*Note: Team Kodi will consider this item however we will not make any promises if it will be included.*

**Comment by FunctionalHacker** [comment]
Just an idea: the proprietary Yatse remote application recommends installing a helper script to Kodi, which adds support for sharing many more sites, including twitch. I believe this is somehow done through youtube-dl so in theory it should support any site that youtube-dl does.

[You can see the code here](https://github.com/Tolriq/repository.yatse.kodi). I don't know how it is licenced though, since the repository doesn't include a licence

I believe if some of this work could be  implemented in Kodi itself, it would make it a better platform for anyone.

**Comment by Dr-cell** [comment]
The idea sounds great to me I also have another company  that likes my
work.This one and another one I worked on, what I would like to discuss is
how much are you willing to pay me for my work so far up to this point and
there could be more ideas in the future you came up with a good one today
on 2,/17/2021

**Comment by Miteirao** [comment]
> Just an idea: the proprietary Yatse remote application recommends installing a helper script to Kodi, which adds support for sharing many more sites, including twitch. I believe this is somehow done through youtube-dl so in theory it should support any site that youtube-dl does.
> 
> [You can see the code here](https://github.com/Tolriq/repository.yatse.kodi). I don't know how it is licenced though, since the repository doesn't include a licence
> 
> I believe if some of this work could be implemented in Kodi itself, it would make it a better platform for anyone.

Yes, please add ResolveURL and youtube-dl support, with manual resolution selection and DASH videos.
